src/inc/quic_platform_posix.h

```diff
@@ -1042,9 +1074,17 @@ CxPlatCqeUserData(
 #include <sys/eventfd.h>
 
 typedef int CXPLAT_EVENTQ;
-#define CXPLAT_SQE int
-#define CXPLAT_SQE_DEFAULT 0
 typedef struct epoll_event CXPLAT_CQE;
+typedef
+void
+(CXPLAT_EVENT_COMPLETION)(
+    _In_ CXPLAT_CQE* Cqe
+    );
+typedef CXPLAT_EVENT_COMPLETION *CXPLAT_EVENT_COMPLETION_HANDLER;
```

Comment:
These can be declared in quic_platform.h?
_posix.h and _winuser.h have same

Reply:
It can't be moved because the definition of CQE is different for each.